File: Treinamento/C/Estrutura_de_decisão/Questão03.c
#include <stdio.h>

int main() {
    char l;

    printf("Digite a primeira letra do sexo desejado: ");
    scanf("%c", &l);

    if (l == 'M' || l == 'm') {
        printf("Sexo digitado: masculino\n"); 
    } else if (l == 'F' || l == 'f') {
        printf("Sexo digitado: feminino\n");
    } else {
        printf("Sexo digitado inválido\n");
    }
}

File: Treinamento/Python/Exercícios/Estrutura_de_decisão/Questão03.py
print('*** Validação de sexo ***')
s = str(input('Digite o sexo: '))

if s.upper() == 'M':
    print('M-Maculino')
elif s.upper() == 'F':
    print('F-Feminino')
else:
    print('Sexo inválido')
